### ISSUE TITLE
process.env is empty object

### ISSUE THREAD

**Issue by blockscoped** [calvinmetcalf/rollup-plugin-node-globals#7]
My rollup config plugins:
```
process.env.MY_PROP = 'test';

const plugins = [
  json(),
  nodeResolve({
    main: true,
    module: true,
    jsnext: true,
    browser: true
  }),
  commonjs({
    include: [
      'node_modules/**'
    ],
    exclude: [
      'node_modules/process-es6/**'
    ],
    namedExports: {
      'node_modules/react/react.js': ['Children', 'Component', 'PropTypes', 'createElement'],
      'node_modules/react-dom/index.js': ['render', 'findDOMNode'],
      'node_modules/react-redux/es/index.js': ['default'],
      'node_modules/redux/es/index.js': ['default'],
      'node_modules/validator/index.js': ['isEmail', 'isNumeric', 'isLength']
    }
  }),
  buble({
    objectAssign: 'Object.assign'
  }),
  globals()
];
```

In my app code when I log `process.env` it has no properties. I would expect to see the `MY_PROP` prop which I'm setting in my config, along with all the other props on `process.env`

Am I missing some setup?

**Comment by calvinmetcalf** [comment]
we don't populate that object currently, you actually don't want to do what you suggested by default, or really ever, because there is a lot of stuff in your env and some of it you'd really not want to have bundled up in your client side code.  it might make sense to add this as an option though to list the properties you want to include from the env.

**Comment by blockscoped** [comment]
Yeah, that makes sense. There's a lot of junk on the node `process.env` that doesn't need to be copied. But I think the idea of and include option is a good one. Probably an array of prop names that you'd like clone. 

Something like:
```
globals({
    include: [
        process: [
            env: ['MY_PROP', 'MY_OTHER_PROP']
        ]
    ]
})
```

**Comment by jedahan** [comment]
Is there a way for the umd to check if it is being required from the commandline, and then proxy the process object?

**Comment by calvinmetcalf** [comment]
if this is running in node...you don't need this library because node provides it all

**Comment by jedahan** [comment]
@calvinmetcalf if I am using rollup to create a single UMD, don't I need this in case its loaded by the browser?

**Comment by calvinmetcalf** [comment]
this is based on how browserify and webpack do it, but they assume their bundles will only be used in a browser, so you may just want to build 2 bundles, one that includes this for the browser and one that doesn't for the server
